### ISSUE TITLE
about accuracy

### ISSUE THREAD

**Issue by shenhaibb** [megvii-model/YOLOF#50]
I use res50-C5
when I don't use the pre-training model, the accuracy is only 59%. After using the pre-training model, the accuracy reaches 80%.
However, my experiment on YOLOv5s is only 75%.
Should I use the pre-training model? What may be the reason for my situation?

**Comment by ytoon** [comment]
Yes, it is better to use the pre-trained model given performance and training time. We suggest you to read this [paper](https://arxiv.org/abs/1811.08883) for a deeper understanding.
